Management/src/ActuatorApi/scaffold/cloudfoundry.py

```diff
@@ -1,16 +1,15 @@
 from pysteel import cloudfoundry
 
-
 def setup(context):
     """
     :type context: behave.runner.Context
     """
     cf = cloudfoundry.CloudFoundry(context)
+    service = 'p.mysql'
+    plan = 'db-small'
+    instance = 'sampleMySqlService'
     # remove previous app
-    app = 'actuator'
+    app = 'actuator-api-management-sample'
     cf.delete_app(app)
     # create service
-    service = 'p.mysql'
-    plan = 'db-small'
-    instance = 'myMySqlService'
     cf.create_service(service, plan, instance)
```

Comment:
```suggestion
from pysteel import cloudfoundry


def setup(context):
    """
    :type context: behave.runner.Context
    """
    cf = cloudfoundry.CloudFoundry(context)
    # remove previous app
    app = 'actuator-api-management-sample'
    cf.delete_app(app)
    # create service
    service = 'p.mysql'
    plan = 'db-small'
    instance = 'sampleMySqlService'
    cf.create_service(service, plan, instance)
```
To sync up with similar files in other samples.